modules/standard/GPU.chpl

```diff
@@ -25,26 +25,30 @@
     This module is unstable and its interface is subject to change in the
     future.
 
+    GPU support is a relatively new feature to Chapel and is under active
+    development.
+
+    For the most up-to-date information about GPU support see the
+    :ref:`technical note <readme-gpu>` about it.
 */
 module GPU
 {
+  pragma "no doc"
   extern proc chpl_gpu_write(const str : c_string) : void;
 
   /*
-     This function is intended to be called from within a GPU
-     kernel and is useful for debugging purposes.
+     This function is intended to be called from within a GPU kernel and is
+     useful for debugging purposes.
 
-     Currently writeln() will make a loop ineligable for
-     gpuization. This function is here as a stopgap measure
-     until we fix that.
+     Currently using :proc:`ChapelIO.write` to send output to ``stdout`` will
```

Comment:
Let's put a `~` before `ChapelIO` so that the name doesn't get displayed, since that module is typically referred to as just "IO Support" and is included by default.